server/instance_cloud_oauth.go

```diff
@@ -41,9 +42,10 @@ type JiraAccessibleResources []struct {
 var _ Instance = (*cloudOAuthInstance)(nil)
 
 const (
-	JiraScopes       = "read:jira-user,read:jira-work,write:jira-work"
-	JiraResponseType = "code"
-	JiraConsent      = "consent"
+	JiraScopes        = "read:jira-user,read:jira-work,write:jira-work"
+	JiraScopesOffline = JiraScopes + ",offline_access"
```

Comment:
Unless we require them to be separate (referencing `JiraScopes` const other than these two lines), I think we should just put `offline_access` in the `JiraScopes` constant and remove the `JiraScopesOffline` constant

Reply:
Yes, the purpose of having these separate is because, we reference `JiraScopes` in the setup instructions for users to add the specified scopes. We do not want users to get confused trying to find `offline_access` in step 5 here:

![setup](https://user-images.githubusercontent.com/1429138/234197630-08f5a53c-1b2c-4905-9150-f651074990ac.png)